yarn-project/noir-contracts/src/contracts/escrow_contract/src/main.nr

```diff
@@ -69,8 +72,7 @@ contract Escrow {
         assert(note.owner == this);
 
         // TODO: Can we dynamically get this selector?
-        let transfer_selector = 0xdcd4c318;
-        let _callStackItem = context.call_private_function(token, transfer_selector, [amount, this, recipient]);
+        let _callStackItem = PrivateTokenContractInterface::at(token).transfer(&mut context, amount, recipient);
```

Comment:
SO HAPPY TO SEE CONTRACT INTERFACES BEING INTRODUCED

Reply:
How was your experience of generating the contract interface, copying it to the correct place, and importing it? Anything we could improve?

Reply:
Copying it to the correct place is a bit inconvenient right now. But not so bad 😃